element_calcium_imaging/scan.py

```diff
@@ -581,18 +605,32 @@ def estimate_nd2_scan_duration(nd2_scan_obj):
             )
 
             self.Field.insert(
-                dict(
-                    key,
-                    field_idx=plane_idx,
-                    px_height=pvscan_info["height_in_pixels"],
-                    px_width=pvscan_info["width_in_pixels"],
-                    um_height=pvscan_info["height_in_um"],
-                    um_width=pvscan_info["width_in_um"],
-                    field_x=pvscan_info["fieldX"],
-                    field_y=pvscan_info["fieldY"],
-                    field_z=pvscan_info["fieldZ"][plane_idx],
-                )
-                for plane_idx in range(pvscan_info["num_planes"])
+                [
+                    dict(
+                        key,
+                        field_idx=0,
+                        px_height=pvscan_info["height_in_pixels"],
+                        px_width=pvscan_info["width_in_pixels"],
+                        um_height=pvscan_info["height_in_um"],
+                        um_width=pvscan_info["width_in_um"],
+                        field_x=pvscan_info["fieldX"],
+                        field_y=pvscan_info["fieldY"],
+                        field_z=pvscan_info["fieldZ"],
+                    )
+                    if pvscan_info["num_planes"] == 1
+                    else dict(
+                        key,
+                        field_idx=plane_idx,
+                        px_height=pvscan_info["height_in_pixels"],
+                        px_width=pvscan_info["width_in_pixels"],
+                        um_height=pvscan_info["height_in_um"],
+                        um_width=pvscan_info["width_in_um"],
+                        field_x=pvscan_info["fieldX"],
+                        field_y=pvscan_info["fieldY"],
+                        field_z=pvscan_info["fieldZ"][plane_idx],
+                    )
+                    for plane_idx in range(pvscan_info["num_planes"])
+                ]
```

Comment:
```suggestion
                    dict(
                        key,
                        field_idx=0,
                        px_height=pvscan_info["height_in_pixels"],
                        px_width=pvscan_info["width_in_pixels"],
                        um_height=pvscan_info["height_in_um"],
                        um_width=pvscan_info["width_in_um"],
                        field_x=pvscan_info["fieldX"],
                        field_y=pvscan_info["fieldY"],
                        field_z=pvscan_info["fieldZ"][plane_idx] if pvscan_info["num_planes"] == 1 else pvscan_info["fieldZ"],
                    )
                    for plane_idx in range(pvscan_info["num_planes"])
```

Would this work?

Reply:
I was thinking about this! If `pvscan_info["num_planes"]` doesn't exist for `num_planes>1`, we can condition the statment in `range`.

Reply:
I'd expect `pvscan_info["num_planes"]` to always exist